bin/celeste.jl

```diff
@@ -32,14 +38,19 @@ The `score-field` subcommand is not yet implemented for the new API.
 function main()
     args = docopt(DOC, version=v"0.1.0", options_first=true)
     Celeste.set_logging_level(args["--logging"])
-    if args["infer-box"]
+    if args["stage-box"] || args["infer-box"]
         ramin = parse(Float64, args["<ramin>"])
         ramax = parse(Float64, args["<ramax>"])
         decmin = parse(Float64, args["<decmin>"])
         decmax = parse(Float64, args["<decmax>"])
-        outdir = args["<outdir>"]
-        Celeste.infer_box_nersc(ramin, ramax, decmin, decmax, outdir)
-    else
+        if args["stage-box"]
+            Celeste.stage_box_nersc(ramin, ramax, decmin, decmax)
+        elseif args["infer-box"]
+            outdir = args["<outdir>"]
+            Celeste.infer_box_nersc(ramin, ramax, decmin, decmax, outdir,
+                                    stage=args["--stage"])
+        end
+    elseif args["infer-field"]
```

Comment:
Done! Just wanted to make sure I wasn't missing anything, and I thought it
might be linked to the SCRATCH issue, though apparently it isn't.

On Wed, Mar 30, 2016 at 2:45 PM Kyle Barbary <notifications@github.com>
wrote:

> In bin/celeste.jl
> <https://github.com/jeff-regier/Celeste.jl/pull/188#discussion_r57969394>:
>
> >          ramin = parse(Float64, args["<ramin>"])
> >          ramax = parse(Float64, args["<ramax>"])
> >          decmin = parse(Float64, args["<decmin>"])
> >          decmax = parse(Float64, args["<decmax>"])
> > -        outdir = args["<outdir>"]
> > -        Celeste.infer_box_nersc(ramin, ramax, decmin, decmax, outdir)
> > -    else
> > +        if args["stage-box"]
> > +            Celeste.stage_box_nersc(ramin, ramax, decmin, decmax)
> > +        elseif args["infer-box"]
> > +            outdir = args["<outdir>"]
> > +            Celeste.infer_box_nersc(ramin, ramax, decmin, decmax, outdir,
> > +                                    stage=args["--stage"])
> > +        end
> > +    elseif args["infer-field"]
>
> Oops, I didn't intend to do that. Should I fix and submit a PR, or is it
> easier for you to do so? (you seem to have the fix in hand...)
>
> —
> You are receiving this because you modified the open/close state.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/jeff-regier/Celeste.jl/pull/188/files/bc9fd33b8eb9f5b1b95131900d25fa826183f075#r57969394>
>